artman/tasks/python_grpc_tasks.py

```diff
@@ -145,3 +151,59 @@ def _copy_and_transform_directories(
                     self.exec_command(['mkdir', '-p', sub_new_src])
                 self._copy_proto(
                     proto, os.path.join(sub_new_src, dest), common_protos)
+
+
+class PythonMoveProtosTask(task_base.TaskBase):
```

Comment:
Why is this done here, instead of when protoc is run? (I.e., can we just have protoc write to the right directory to begin with?)

Reply:
I will check on this.